lightning/src/util/persist.rs

```diff
@@ -11,13 +11,17 @@
 //! [`ChannelManager`]: crate::ln::channelmanager::ChannelManager
 //! [`NetworkGraph`]: crate::routing::gossip::NetworkGraph
 
+use alloc::sync::Arc;
```

Comment:
The diff of this commit is hard to review. Not sure if improvement is possible though.